src/util.py

```diff
@@ -28,14 +28,12 @@ def prepare_path_docker(orig_path: PurePath) -> str:
     return prepared_path
 
 
-def parse_config(config_file):
+def process_config(config):
     """
-    Parse the config file and return the full yaml structure as well as processed portions
+    Process the dictionary config and return the full yaml structure as well as processed portions
     @param config_file: the path to the config file
```

Comment:
This argument description can be updated.  Now the parameter `config` is the configuration loaded by Snakemake from the config file and any command line arguments.